### DISCUSSION TITLE
Android markdown rendering issue

### DISCUSSION THREAD

**Discussion by qw-in** [community/community#35724] [Mobile] [answered]
This md won't render at all when viewing the repo. Navigating to the readme.md file directly results in very broken results

https://github.com/vercel/satori/blob/main/README.md

![Screenshot_2022-10-10-16-43-31-59_320a9a695de7cdce83ed5281148d6f19.jpg](https://user-images.githubusercontent.com/19194187/194968493-c7160c1e-ab01-4b3c-a575-e839e29c0fb3.jpg)
![Screenshot_2022-10-10-16-43-19-53.jpg](https://user-images.githubusercontent.com/19194187/194968499-d3a24c87-43fc-42e9-b749-3abb9080fba2.jpg)

**Comment by stevepopovich** [comment, marked as answer]
Hi Quinn, yup this looks a little wonky! I file a bug for the team to look at this. Thanks for reaching out!

**Comment by aconsuegra** [comment]
Hi @qw-in, thanks for reporting the bug. May I ask which device and Android version are you using? Thanks!

**Reply by qw-in** [reply]
Oneplus 8 pro - android 12

**Comment by stevepopovich** [comment]
Hi @qw-in are you still seeing this issue? We no longer see the issue

**Reply by qw-in** [reply]
Fixed for me!

**Reply by stevepopovich** [reply]
Great!
